website/addons/wiki/templates/edit.mako

```diff
@@ -219,7 +219,7 @@
   <div class="modal-dialog">
     <div class="modal-content">
       <div class="modal-header">
-        <h3 class="modal-title">The permissions for this page have changed</h3>
```

Comment:
I agree with @haoyuchen1992 . I'm going to close this for now until we have more well-established guidelines for this case.